src/rules/memberOrderingRule.ts

```diff
@@ -487,3 +555,34 @@ function nameString(name: ts.PropertyName): string {
             return "";
     }
 }
+/**
+ * Returns the last element of an array. (Or undefined).
+ */
+function arrayLast<T>(array: ArrayLike<T>): T {
+    return array[array.length - 1];
+}
+
+/**
+ * Array.prototype.findIndex, but the last index.
+ */
+function arrayFindLastIndex<T>(
+    array: ArrayLike<T>,
+    predicate: (el: T, elIndex: number, array: ArrayLike<T>) => boolean,
+): number {
+    for (let i = array.length; i-- > 0;) {
+        if (predicate(array[i], i, array)) {
+            return i;
+        }
+    }
+    return -1;
+}
+
+/**
+ * Applies a Replacement to a part of the text which starts at offset.
```

Comment:
this comment does not match the impl. it returns a string??

Reply:
It's just a slightly modified version of Replacement.apply https://github.com/palantir/tslint/blob/master/src/language/rule/rule.ts#L204, which also returns a string.